pontoon/sync/management/commands/import_pootle.py

```diff
@@ -113,13 +113,12 @@ def handle_project_locale(self, project, locale):
         users_dict = {}
         users = User.objects.filter(email__in=emails)
         missing_users_list = []
-
+        locale_translators_map = dict(Locale.objects.values_list('code', 'translators_group'))
         for u in users:
             users_dict[u.email] = u
             if not u.has_perm('base.can_localize'):
```

Comment:
You should use the new API here:
u.has_perm('base.can_translate_locale', locale)